e2e/test/scenarios/dashboard-filters/dashboard-filters-date.cy.spec.js

```diff
@@ -106,12 +107,19 @@ describe("scenarios > dashboard > filters > date", () => {
     toggleRequiredParameter();
     dashboardSaveButton().should("be.disabled");
     dashboardSaveButton().realHover();
-
     cy.findByRole("tooltip").should(
       "contain.text",
       'The "Month and Year" parameter requires a default value but none was provided.',
     );
 
+    // Can't close sidebar without a default value
+    dashboardParametersDoneButton().should("be.disabled");
+    dashboardParametersDoneButton().realHover();
```

Comment:
This is the nittiest nit ever, but just FYI this also works:
```js
dashboardSaveButton().should("be.disabled").realHover();
```

Reply:
Thanks, I'll change that in one of the next commits/PRs so that I don't have to wait for the CI again.